tests/main/views/test_users.py

```diff
@@ -1166,6 +1167,83 @@ def test_can_update_phone_number(self):
         assert data['phoneNumber'] == '0800666666'
 
 
+class TestAdminManagerUpdate(BaseApplicationTest):
+    """
+    Test the log output for the admin-manager password reset.
+    The message should only be output when and admin manager changes their password
+    """
+    admin_manager_warning_log_message = mock.call(
+        "{code}: Password reset requested for {user_role} user '{email_hash}'",
+        extra={
+            'code': 'update_user.password.role_warning',
+            'email_hash': 'W-jA20fzplHyOKk82Io1EyjW_l4X25Min3NRyU0hgiY=',
+            'user_role': 'admin-manager'
+        }
+    )
+
+    def setup(self):
+        with mock.patch('flask.app.create_logger', return_value=mock.Mock(spec=Logger('flask.app'), handlers=[])):
```

Comment:
Ahh I've not seen this trick before - do we use it elsewhere?

Reply:
Yeah it's one of the ways we have in utils. It's good in a way because it's completely handler agnostic

https://github.com/alphagov/digitalmarketplace-utils/blob/master/tests/conftest.py#L43